### PR TITLE
Implemented select/deselect option for tags dropup in graph editor

### LINKED ISSUE #98: Add a select all/deselect all for tags
If you have a very long list of tags, it becomes unwieldy to select or deselect all the tags on the list. 

The ask is for there to be an ability to easily do so. 

Please let me know if additional clarification is needed.

### PR DESCRIPTION
**Pull request by Mr-Nobody99**
resolves #98

### Description
After seeing the open issue for a select/deselect all option for tags in the graph editor, I thought that makes a lot of sense.
So I implemented it, the changes are in the graph-launcher. I added a new list item in the html and some conditions in onItemSelect() to handle the change.

### Checklist
 - [x] I have signed the [CLA](https://docs.getdbt.com/docs/contributor-license-agreements)
 - [x] I have generated docs locally, and this change appears to resolve the stated issue
 - [x] I have updated the `CHANGELOG.md` and added information about my change to the "dbt next" section.

### CODE REVIEW

File: src/app/components/graph/graph-launcher.html
Comment (drewbanin) [inline src/app/components/graph/graph-launcher.html:176]: by treating this as a regular item that can be selected, we can find ourselves in some funky states! Check this example out, for instance:

Screen Shot 2020-09-25 at 2 00 47 PM

Instead, I think we might just want to add a new method in the scope called something like `toggleSelectAll('tags')`. This method would either unselect all tags, or make all tags selected depending on the state. I think that would help clean up some of the logic added in `graph-launcher.js ` too!

Let me know if you have any thoughts or questions about an approach like this one :)

**Review by drewbanin — COMMENTED** [review]
Cool PR @Mr-Nobody99 - i like this a lot!

Couple of quick things:
 - When all tags are unselected, the select state shows `select_all`. I dropped a comment inline in the code about how I think we could resolve this, but curious to hear what you're thinking too.

Screen Shot 2020-09-25 at 1 51 42 PM

- It probably makes sense to add this same component to the `resources` and `packages` selectors too! You think there's a straightforward way to this option available on each of the dropdowns in the DAG view?

Nice work so far!

File: src/app/components/graph/graph-launcher.html
Comment (Mr-Nobody99) [reply src/app/components/graph/graph-launcher.html:177]: sure that's no problem, I left it as default link color with the thought that that might distinguish it a bit from the rest of the list. But I think just doing bold is good as well.

File: src/app/components/graph/graph-launcher.html
Comment (Mr-Nobody99) [reply src/app/components/graph/graph-launcher.html:176]: I definitely see your point, I can change it to a separate function. That is what I had considered at first, I thought the current was just less intrusive.

File: src/app/components/graph/graph-launcher.html
Comment (drewbanin) [inline src/app/components/graph/graph-launcher.html:118]: Just curious - why do `allSelected ? false : true` instead of `!allSelected`?

**Comment by drewbanin** [comment]
This is really excellent! Thanks for making those changes - I think this works really well now. Can you just:
1. Check out my latest comment
2. Fix the merge conflicts in the changelog?

After that, this should be ready to merge!

File: src/app/components/graph/graph-launcher.html
Comment (Mr-Nobody99) [reply src/app/components/graph/graph-launcher.html:118]: No particular reason I didn't think of that, but that is a bit cleaner I'll change it.

File: CHANGELOG.md
Comment (drewbanin) [inline CHANGELOG.md:4]: can you just add yourself as a contributor in the existing section below? Looks like we have two contributor sections for the 0.19.0 release now :)

**Review by drewbanin — APPROVED** [review]
Approving this pending the tiniest of changes in the changelog. Nice work on this one!

File: CHANGELOG.md
Comment (Mr-Nobody99) [reply CHANGELOG.md]: Ahh, yep just corrected it thanks.

**Review by drewbanin — APPROVED** [review]
Merging! This will ship in dbt v0.19.0 :)

**Comment by Mr-Nobody99** [comment]
Woot! thanks a lot

**Pull request merged**
